.github/workflows/sonar.yaml

```diff
@@ -7,35 +7,39 @@ jobs:
     name: SonarCloud Trigger
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
-
-    - name: Setup node 13.x
-      uses: actions/setup-node@v1
-      with:
-        node-version: 13.x
-
-    - name: Get npm cache directory
-      id: npm-cache
-      run: |
-        echo "::set-output name=dir::$(npm config get cache)"
-
-    - uses: actions/cache@v1
-      with:
-        path: ${{ steps.npm-cache.outputs.dir }}
-        key: ubuntu-latest-node-${{ hashFiles('**/package-lock.json') }}
-        restore-keys: |
-          ubuntu-latest-node-
-
-    - name: Install dependencies
-      run: |
-        npm ci
-
-    - name: Build & Test
-      run: |
-        npm run test
-
-    - name: SonarCloud Scan
-      uses: sonarsource/sonarcloud-github-action@master
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        SONAR_TOKEN: ${{ secrets.SONARCLOUD_TOKEN }}
+      - uses: actions/checkout@v1
+
+      - name: Setup node 13.x
```

Comment:
What do you think to increase this version to be compatible with .nvmrc?

```diff
-13.x
+14.x
```

Reply:
Good observation. Done!